### DISCUSSION TITLE
Re-size or scroll the Design tree pane?

### DISCUSSION THREAD

**Discussion by kmdelucaNG** [Budibase/budibase#3587] [Help]
I have Budibase v0.9.168 and Chrome 92.

There doesn't seem to be a way to resize or horizontally scroll the Design tree pane - so components are not visible once you get to a certain indent level.  Is there another way to do that so I can see all the indented components?
![DesignPane](https://user-images.githubusercontent.com/86023779/143932006-7e1733f5-8d70-4716-9dba-6a80b119b5a6.jpg)

**Comment by mjashanks** [comment]
Thanks for reporting this, we'll look into it!

This is logged as an issue: https://github.com/Budibase/budibase/issues/2159

**Reply by gvlx** [reply]
I left a hack as a comment here: https://github.com/Budibase/budibase/issues/2159#issuecomment-968135525
